crates/re_data_ui/src/pinhole.rs

```diff
@@ -11,6 +11,17 @@ impl DataUi for PinholeProjection {
         verbosity: UiVerbosity,
         query: &re_arrow_store::LatestAtQuery,
     ) {
+        if verbosity == UiVerbosity::Small {
+            // See if this is a trivial pinhole, and can be displayed as such:
+            let fl = self.focal_length_in_pixels();
+            let pp = self.principal_point();
+            if *self == Self::from_focal_length_and_principal_point(fl, pp) {
+                ui.label(format!("Focal length: {fl}\nPrincipal point: {pp}"))
+                    .on_hover_ui(|ui| self.data_ui(ctx, ui, UiVerbosity::Reduced, query));
```

Comment:
focal length is most of the time same on X and Y. I think the ui should tighten this further in that case

Reply:
Good point!

![image](https://github.com/rerun-io/rerun/assets/1148717/537d2d5b-6b11-47a1-82c7-4c5b054b458e)